### PR TITLE
Alphabetical Search

### PR DESCRIPTION
**Pull request by rexdildo**
Instead of just sorting the keywords this adds another layer that lets you choose by alphabetical letter upfront. Usefull if you have quite a few keywords.

### CODE REVIEW

**Comment by rexdildo** [comment]
I can make a pullrequest for cumination too.  Perhaps make a development branch if you don't want all the source files in the master branch.

File: plugin.video.uwc/resources/lib/utils.py [hunk 1]
```diff
@@ -824,6 +825,16 @@ def searchDir(url, mode, page=None):
         pass
     xbmcplugin.endOfDirectory(addon_handle)
 
+@url_dispatcher.register('920', ['url', 'channel'], ['keyword'])
```
Comment (dobbelina) [inline plugin.video.uwc/resources/lib/utils.py:828]: @rexdildo 
It crashes on my Windows, Get an error here:
```
File "C:\Users\Test\AppData\Roaming\Kodi\addons\plugin.video.uwc\resources\lib\sites\hentaidude.py", line 25, in <module>
                                                @utils.url_dispatcher.register('920')
                                              File "C:\Users\Test\AppData\Roaming\Kodi\addons\plugin.video.uwc\resources\lib\url_dispatcher.py", line 47, in decorator
                                                raise Exception(message)
                                            Exception: Error: <function Main at 0x000001C75BC6BEB8> already registered as 920
```
Thanks for the contribution.
You can make pull requests to the main branch for Cumination.
Our coders will review it and incorporate it in the next release if they don't have any issues with it.

**Comment by rexdildo** [comment]
that happens when you edit code on github without testing. lol.  should work now, i will post a pull request for cuminatiion (after testing this time :)

**Comment by rexdildo** [comment]
the main branch of cumintaion still only contains the zip files and some meta files, the folder with the resources is missing. that's what i meant when i said the source files are missing.  so there is no way to do a pull request,

**Comment by dobbelina** [comment]
@rexdildo 
Upload the files with the correct folder structure to your fork, then you can make a pull request.
Not all files, just the one's you have changed.

**Comment by dobbelina** [comment]
@rexdildo @Gujal00 @12asdfg12 
I have uploaded utils.py as i assume that's the file you want to edit.
Makes it easier for our coders to see what has been changed.

**Comment by rexdildo** [comment]
actually it's 2 files to change. basics.py too. if you want people to contribute than it would be helpfull to have a proper develoment environment.

**Comment by dobbelina** [comment]
@rexdildo 
I have started a discussion with our developers regarding this issue.
I'll keep you posted.

**Pull request merged**